tests/Integration/HasSlugTest.php

```diff
@@ -269,4 +269,12 @@ public function getSlugOptions(): SlugOptions
         $model->save();
         $this->assertEquals('guete-nacht', $model->url);
     }
+
+    /** @test */
+    public function it_can_handle_uuid()
+    {
+        $model = UuidTestModel::create(['id' => '6dae40fa-cae0-11e7-80b6-8c85901eed2e', 'name' => 'test name']);
+
+        $this->assertEquals('test-name', $model->url);
```

Comment:
I don't understand what this test is actually testing.

Reply:
This is the test I wrote to check if by using a uuid on a sqlite database still worked after applying the changes (I get the redundantie of the test).